M/MDLovoFit/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MDLovoFit"
+version = v"20.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/m3g/MDLovoFit.git", "e320c2a503c9bd88fe2dc102f2abfd0b2f5d4cce")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/MDLovoFit
+install_license LICENSE
+cmake .
```

Comment:
Please read the documentation, you're missing all the important arguments to cmake: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds

Reply:
Thanks Mosè. I think and hope that now everything is correct.